src/modules/site-v2/templates/_scripts/utils.js

```diff
@@ -44,3 +44,28 @@ function form_data_to_object(form_id) {
   });
   return data;
 }
+
+
+function force_download(el, url) {
```

Comment:
Some thoughts on the utility interface: 

Looking at how this is being used, and maybe it can be simplified -- if this is being used as an event handler callback, you can keep the `href` populated in the `<a/>` element, and just use `onclick=force_download`. This will call your function with a parameter `event` that contains a reference to the <a/> inside of `event.target` where you can find the `event.target.href`. At that point, since you have access to the `event` itself in the callback, you can use event.stopPropagation(), event.preventDefault() to avoid having it follow the link. 

on the other hand, if we want a `pure` download function, we should only pass the `url` as the only parameter, and then create/destroy the necessary <a/> tag dynamically (add to the body, click, remove from the body).